src/Components/test/E2ETest/ServerExecutionTests/TestSubclasses.cs

```diff
@@ -37,9 +36,12 @@ public ServerEventTest(BrowserFixture browserFixture, ToggleExecutionModeServerF
     public class ServerInteropTest : InteropTest
     {
         public ServerInteropTest(BrowserFixture browserFixture, ToggleExecutionModeServerFixture<Program> serverFixture, ITestOutputHelper output)
-            : base(browserFixture, serverFixture.WithServerExecution(), output)
+            : base(browserFixture, serverFixture.WithServerExecution().WithAdditionalArguments(GetAdditionalArguments()), output)
         {
         }
+
+        private static string[] GetAdditionalArguments() =>
+            new string[] { "--circuit-detailed-errors", "true" };
```

Comment:
Instead of going to this trouble to pass `--circuit-detailed-errors` just for this one test class (and having machinery for passing through additional options), how about just hard-coding the option into `TestServer`'s `Startup.cs`?

Unless we're testing how the behavior varies when the option is disabled (and it doesn't seem that we are), there's no strong benefit from passing through that info from here. Also we probably want to default to "detailederrors:true" for the test server anyway, since it's generally useful for debugging.

Reply:
We are.

JSInterop tests test the value with detailed errors on, while there is another E2E test specifically for the server-side default.

If we were to remove this "Machinery" we could test only one or the other which would:
1) Complicate the JS Interop tests by having to differentiate from client/server.
2) Skip testing an important default value.